site/content/getting-started/_index.md

```diff
@@ -14,57 +13,29 @@ The quickstart guide uses all default settings. No additional configuration is r
 
 # What is Contour?  
 
-Contour is an open source Kubernetes ingress controller providing the control plane for the Envoy edge and service proxy.​ Contour supports dynamic configuration updates and multi-team ingress delegation out of the box while maintaining a lightweight profile.  
-
-# Philosophy
-- Follow an opinionated approach which allows us to better serve most users
-- Design Contour to serve both the cluster administrator and the application developer
-- Use our experience with ingress to define reasonable defaults for both cluster administrators and application developers.
-- Meet users where they are by understanding and adapting Contour to their use cases  
-
-See the full [Contour Philosophy][5] page.
+Contour is an Ingress controller for Kubernetes that works by deploying the [Envoy proxy][26] as a reverse proxy and load balancer.
+Contour supports dynamic configuration updates out of the box while maintaining a lightweight profile.
 
-# Why Contour?
-Contour bridges other solution gaps in several ways:
-- Dynamically update the ingress configuration with minimal dropped connections
-- Safely support multiple types of ingress config in multi-team Kubernetes clusters
-- Improve on core ingress configuration methods using our HTTPProxy custom resource
-- Cleanly integrate with the Kubernetes object model  
 
 # Quickstart
 This quickstart guide installs Contour on a kubernetes cluster with a web application workload.
-1. Set up a kubernetes environment
-1. Install a Contour service
+1. Set up a Kubernetes environment
+1. Install a Contour & Envoy
 1. Install a kuard workload
 
 
-## 1. Set up a kubernetes environment
-This procedure uses Docker and kind to deploy a kubernetes cluster. If you already have a cluster available, skip to step 2  – Install a Contour service.  
-
-
-### Install kind:
-
-See the download and install instructions for kind [here][22].
-
-Verify kind is installed by running:
-
-```yaml
-$ kind
-```
-You should see a list of kind commands.  
-
-### Install Docker:  
+## Set up a Kubernetes environment
```

Comment:
I wonder if we should move the "how to create a Contour-compatible KinD cluster" instructions to a separate page entirely. Then, the quickstart here could say it assumes you have a cluster up and running (which I'd think most folks would), and if not, here's a link to how to set up a local KinD cluster. Benefit being that we get to "install Contour" faster on this page.

Reply:
+1 to this idea, it makes it clearer that you don't *have* to install kind, just have a cluster.